tools/nut-scanner/nutscan-snmp.h

```diff
@@ -21,6 +21,8 @@
 #ifndef DEVSCAN_SNMP_H
 #define DEVSCAN_SNMP_H
 
+#include <stddef.> //define NULL
```

Comment:
so is it without an extension or with a dot-h? ;)
also while the fix is reasonable (I hit the issue too) I think it better belongs in some other include-file or source-file? Then again, fenced by standard ifdef's, it is not a huge overhead... so maybe ok here.